src/effects/backends/builtin/echoeffect.cpp

```diff
@@ -118,6 +120,14 @@ void EchoEffect::loadEngineEffectParameters(
     m_pTripletParameter = parameters.value("triplet");
 }
 
+float averageSampleEnergy(std::span<const CSAMPLE> delay_buffer) {
```

Comment:
Energy is voltage * current. Here you use only voltage. 
```suggestion
float averageSampleLevel(std::span<const CSAMPLE> delay_buffer) {
```

Reply:
We collect here such sample functions that are reusable together with a hint that indicates if the function is auto vectorized.

This is probably already the case here, but worth to double check. 

https://github.com/mixxxdj/mixxx/blob/ae2dc103bc991f1c3fd182dbcc2fa8bf535e8f77/src/util/sample.cpp#L463